tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -127,6 +127,20 @@ public void Should_not_be_able_to_get_types_from_assembly_that_does_not_referenc
             catalogue.GetAvailableTypes().Should().NotContain(typeof(string));
         }
 
+        [Fact]
+        public void Should_warn_if_some_types_cannot_be_loaded_from_external_assembly()
+        {
+            // Arrange
+            var catalogue = new TypeCatalogue();
+
+            // Act
+            string output = CaptureConsoleOutput(
+                () => catalogue.Load(new[] { this.externalAssemblyGenerator.AssemblyOriginalPath }));
+
+            // Assert
+            output.Should().Match("*Warning: FakeItEasy failed to get some types from assembly 'FakeItEasy.ExtensionPoints.External*' while scanning for extension points. Some IArgumentValueFormatters, IDummyFactories, and IFakeOptionsBuilders in that assembly might not be available.*");

```

Comment:
Ah, but it's *not* named in this case... The exception doesn't contain that information. The errors related to types in `DynamicProxyGenAssembly2` did contain the type name, because they were `TypeLoadException`s. In this case it's a `FileNotFoundException`.